server/config/summaries/place_summaries.json

```diff
@@ -0,0 +1,152 @@
+ },
+ "geoId/06": {
+  "summary": "California is the most populous state in the United States with a population of over 39 million people. The state's population has increased by over 27 million people since 1900. The median individual income in California was $27,355 in 2011 and rose to $36,281 in 2021. California's median age is 37.0 years, which is ranked 44th out of 52 states in the United States. The median age has increased by 1.9 years from 2011 to 2021."
+ },
+ "geoId/48": {
+  "summary": "Texas is the second most populous state in the United States of America. The population of Texas has increased from 3055000 in 1900 to 29527941 in 2021. The median individual income in Texas increased from 25554 in 2011 to 33644 in 2021. Texas is ranked 30 of 52 states in United States of America. The median age in Texas has increased from 33.5 in 2011 to 35.0 in 2021. Texas is ranked 49th out of 52 states in the United States of America for median age."
+ },
+ "geoId/12": {
+  "summary": "Florida is the 3rd most populous state in the United States with a population of 21.8 million people. The median individual income in Florida was $31,169 in 2021, up from $25,014 in 2011. Florida has the 6th highest median age in the United States at 42.3 years, up from 40.5 years in 2011."
+ },
+ "geoId/36": {
+  "summary": "New York is the 4th most populous state in the United States with a population of 19.83 million. The population has increased from 7.28 million in 1900. New York's median individual income increased from 28670 in 2011 to 37469 in 2021. New York is ranked 15th in the United States of America for median income. The median age of New York is 39.2 years, which is ranked 21st out of 52 states in the United States. The median age has increased by 1.4 years from 2011 to 2021."
+ },
+ "geoId/42": {
+  "summary": "Pennsylvania is the 5th most populous state in the United States with a population of 12.9 million people. The median age in Pennsylvania is 40.8, and the median individual income is $34,229. Pennsylvania is ranked 25th in the United States for median individual income."
+ },
+ "geoId/38": {
+  "summary": "North Dakota is a state in the United States with a population of 774,948 in 2021. It is ranked 48th in population out of the 50 states. The median individual income in North Dakota increased from $25,925 in 2011 to $37,861 in 2021, and the state is ranked 11th in the United States for median individual income. North Dakota has a median age of 35.2 years, which is ranked 48th out of 52 states in the United States. The median age has decreased slightly from 37.1 years in 2011 to 35.2 years in 2021."
+ },
+ "geoId/02": {
+  "summary": "Alaska is the 49th most populous state in the United States with a population of 732,673. The median individual income in Alaska was $37368 in 2021, ranking 13th in the United States. Alaska has the 50th highest median age in the United States at 35.0."
```

Comment:
Error: "49th most populous"

Reply:
yes, i need to spend more time tuning the smaller places. there are more of these, like "1028th of 1028 populous ..."